python-package/xgboost/data.py

```diff
@@ -165,7 +165,7 @@ def _is_scipy_coo(data: DataType) -> bool:
 
 
 def _is_numpy_array(data: DataType) -> bool:
```

Comment:
Perhaps now a more appropriate name for this is `_is_numpy_array_like`?

Reply:
Thank you for the suggestion, renamed.